pkg/cmd/cli/cmd/request_project.go

```diff
@@ -113,23 +116,36 @@ func (o *NewProjectOptions) Run() error {
 		return err
 	}
 
-	if o.ProjectOptions != nil {
-		o.ProjectOptions.ProjectName = project.Name
-		o.ProjectOptions.ProjectOnly = true
-		o.ProjectOptions.SkipAccessValidation = true
+	if !o.SkipConfigWrite {
+		if o.ProjectOptions != nil {
```

Comment:
reading more carefully, couldn't `SkipConfigWrite` cause us to avoid creating the `ProjectOptions` in `complete`, which would prevent us from running `RunProject`, which would prevent the config write? 

If you did that, then our nesting here stays the same, the existing `Fprintf` shoudl already have been under the existing `if` block, and you're just adding an `else` saying you can switch if you want?


```go
if projectoptions != nil{
    // do existing working
    printf "you can add applications"
} else{
    printf "to switch to this project"
}
```